salt/modules/redismod.py

```diff
@@ -69,6 +69,17 @@ def _sconnect(host=None, port=None, password=None):
     return redis.StrictRedis(host, port, password=password)
 
 
+def _get_kwargs_opts(**options):
```

Comment:
can we use this opportunity to add `salt['config.get']()`to the options here so that the redis connection can be set in the pillar data?